organizations/organization-roles.skos.ttl

```diff
@@ -8,20 +8,21 @@
   skos:prefLabel "Meemoo organization Roles thesaurus" ;
   skos:hasTopConcept <Staff> .
 
-<Staff> a org:Role, skos:Concept;
```

Comment:
Heb je dit verwijderd omdat dit ook door een reasoner kan afgeleid worden? Klopt helemaal, maar omdat er bij sommige tools niet altijd een reasoner voorhanden is hadden we dit laten staan. Nu OK om dit door te voeren; de thesauri worden toch nog herbekeken. Maar we gaan moeten nadenken over de afhankelijkheid op een reasoning stap

Reply:
Ik had het verwijderd omdat het ook met een reasoner kan worden afgeleid. Maar als het bijvoorbeeld nodig is om documentatie te genereren is het niet ongebruikelijk om het te laten staan. In principe is er een reasoner voorhanden in TriplyDB met JENA waardoor we toch alles kunnen afleiden maar het is wel een extra stap. 
@kathrin